src/template/config.terrain.json

```diff
@@ -8,22 +8,38 @@
       }
     }
   },
-  "mainnet": {
-    "_connection": {
-      "chainID": "columbus-5",
-      "URL": "https://lcd.terra.dev"
+  "mainnet": {    
+    "phoenix-1": {
+      "chainID": "phoenix-1",
+      "lcd": "https://phoenix-lcd.terra.dev",
+      "gasAdjustment": 1.75,
+      "gasPrices": { "uluna" : 0.015 },
+      "prefix": "terra"
+    },
+    "osmosis-1": {
+      "chainID": "osmosis-1",
+      "lcd": "https://osmosis-1.terra.dev",
+      "gasAdjustment": 1.75,
+      "gasPrices": { "uosmo": 0.025 },
+      "prefix": "osmo"
     }
   },
   "testnet": {
-    "_connection": {
+    "pisco-1": {
       "chainID": "pisco-1",
-      "URL": "https://pisco-lcd.terra.dev"
+      "lcd": "https://pisco-lcd.terra.dev",
+      "gasAdjustment": 3.5,
+      "gasPrices": { "uluna": 0.015 },
+      "prefix": "terra",
+      "name": "terra"
     }
   },
   "localterra": {
```

Comment:
since we plan to support multichain, maybe we can rename this to `localnet`? this is probably a nit, since the inner `localterra` is the real chain id used for localnet, outer naming doesn't matter